<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrum Analyzer Tool: Video Processor</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="loadVideoWidget">
   <widget class="QPushButton" name="loadVidBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Press to Select Video File</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note: File Type Must Be mp4 or mkv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csvFilepathBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Press to Change CSV Filepath</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="csvFilepathLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
